bin/www/codePush.js

```diff
@@ -25,6 +25,11 @@ var CodePush = (function () {
         cordova.exec(installSuccess, errorCallback, "CodePush", "restartApplication", []);
     };
     CodePush.prototype.reportStatus = function (status, label, appVersion, deploymentKey, previousLabelOrAppVersion, previousDeploymentKey) {
+        if ((!label || label === previousLabelOrAppVersion) &&
+            appVersion === previousLabelOrAppVersion &&
+            deploymentKey === previousDeploymentKey) {
```

Comment:
shouldn't this be: 
`((!label && appVersion === previousLabelOrAppVersion) || (label === previousLabelOrAppVersion)) && deploymentKey === previousDeploymentKey`